src/governance.rst

```diff
@@ -0,0 +1,172 @@
+Governance
+==========
+This document outlines the policies and procedures that manage the conda-forge
+community. This document recognizes that, while packaging is deeply important
+to all of us, conda-forge is an all-volunteer organization. Members of the
+core, any team, or external contributors choose to participate and may
+choose to leave at any time for any reason or for no reason. We deeply
+appreciate all good faith contributions.
+
+Code of Conduct
+---------------
+Conda-forge adheres to the
+`Numfocus Code of Conduct <https://www.numfocus.org/code-of-conduct>`_.
+
+Teams & Roles
+-------------
+Here are defined the primary teams participating in conda-forge activities.
+
+* **core:** The core team is the governing body over the entire conda-forge
+  organization. Members of the core team have full rights over all conda-forge
+  repositories. Members of core are the face of the project, and are responsible
+  for officially interfacing with external communities, organizations, non-profits,
+  and companies. The core team may create new sub-teams, as appropriate.
+  Each member of core is entitled to one vote on all elected matters.
+* **staged-recipes:** The staged-recipes team administers the staged-recipes
+  repository and is responsible for the review and creation of new feedstocks.
+* **maintainers:** A maintainer is an individual responsible for the management
+  of one or more feedstock repositories and packages. Maintainers have the ability
+  to merge pull requests into the feedstock of only the packages they maintain.
+* **external contributors:** This group encompasses all others who are not on
+  core, part of staged-recipes, or maintainers. This includes first time
+  contributors, collaborators, and funders. They have no special rights within
+  the conda-forge organization itself.
+
+Sub-Teams
+---------
+The core team may elect to create new sub-teams for managing the daily business
+of the organization. While sub-teams may have non-core members, every sub-team
+must have at least one core team member at all times. If a sub-team fails to
+have  a core team member for more than 1 week, that team is considered to be
+dissolved. A new sub-team would need to be established by the core in order to
+reinstate the activity.
+
+Sub-teams have a charter that is either *dynamic* or *static*.
+
+* A *dynamic* charter means that the sub-team is self-organizing, with respect
+  to its own internal policies, procedures, and membership. A sub-team may choose
+  to modify its membership independent of the core team. For example, a
+  Google Summer of Code team could be a good candidate for a dynamic charter.
+  Alternatively, language-based maintenance teams also have a dynamic charter.
```

Comment:
Any thoughts on core's role with dynamic teams? Are they like the team's BDFL or something else?

Reply:
From our discussion earlier, it sounds like this core member exists to form a link between core and the subteam for communication purposes. They may not necessarily even lead the team.